core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -484,14 +487,14 @@ class LogCleanerTest {
 
     // first time through the control batch is retained as an empty batch
     // Expected State: [{Producer1: EmptyBatch}], [{2}, {3}]
-    var dirtyOffset = cleaner.doClean(LogToClean(tp, log, 0L, log.activeSegment.baseOffset), deleteHorizonMs = Long.MaxValue)._1
+    var dirtyOffset = runTwoPassClean(cleaner, LogToClean(tp, log, 0L, log.activeSegment.baseOffset), currentTime = largeTimestamp)
```

Comment:
Could we add the following comment?

"In the first pass, the deleteHorizon for the commit marker is set. In the second pass, the commit marker is removed, but the empty batch is retained for preserving the producer epoch."